File: articles/donald-knuth-and-the-art-of-computer-programming/fortran/growth_comparison.f90
program growth_comparison
  implicit none
  real :: n
  n = 1000.0
  print *, "log2_n=", log(n) / log(2.0)
  print *, "n=", n
  print *, "n_log2_n=", n * log(n) / log(2.0)
  print *, "n_squared=", n * n
end program growth_comparison

File: articles/donald-knuth-and-the-art-of-computer-programming/java/GrowthComparison.java
public class GrowthComparison {
    public static void main(String[] args) {
        double n = 1000.0;
        System.out.printf("log2_n=%.6f%n", Math.log(n) / Math.log(2));
        System.out.printf("n=%.0f%n", n);
        System.out.printf("n_log2_n=%.6f%n", n * (Math.log(n) / Math.log(2)));
        System.out.printf("n_squared=%.0f%n", n * n);
    }
}
